apps/webapp/web/static/js/docs/DocPage.js

```diff
@@ -0,0 +1,22 @@
+// file: docs/DocPage.js
+import React from 'react';
+import mdlUpgrade from 'lib/mdlUpgrade';
+import material from 'material-design-lite/material.css';
+import styles from './styles.css';
+import ReactMarkdown from 'react-markdown';
+
+class DocPage extends React.Component {
+
+  render() {
+    return (
+      <div styleName="mdl-grid">
+        <div styleName="mdl-cell mdl-cell--12-col">
+          <h3>{this.props.path}</h3>
```

Comment:
In this component, when `componentDidMount()`, you need to fetch the content from `this.props.path`. You need to implement an **action** in `DocController`. In REST an action of similar nature is typically called `show` and given a `:doc_path` in params it should return a content that you can render in your component.